byoc: update onboarding content (#23393)
pingcap/docs: tidb-cloud/byoc/byoc-prepare-environment-aws.md

To meet compliance requirements, TiDB Cloud BYOC integrates with a customer-provided PCA to issue identity certificates for data nodes using your enterprise's own domain. This ensures that the Root of Trust for all encrypted communications remains strictly within your organization's control.

> Therefore, you must prepare a valid subordinate CA in the deployment region. The CA ARN that you provide to TiDB Cloud must be the ARN of the subordinate CA. To create and activate the subordinate CA, you need an active root CA to sign its certificate.

Reviewer: Is the subordinate CA still a MUST? And can it not directly use a Root CA instead?

Author: a MUST for security

> Therefore, you must prepare a valid subordinate CA in the deployment region. The CA ARN that you provide to TiDB Cloud must be the subordinate CA ARN. To create and activate the subordinate CA, you need an active root CA to sign the subordinate CA certificate.

Reviewer 2: ```suggestion
Therefore, you must prepare a valid subordinate CA in the deployment region. The CA ARN that you provide to TiDB Cloud must be the ARN of the subordinate CA. To create and activate the subordinate CA, you need an active root CA to sign its certificate.
```

> The subordinate CA certificate must be valid for at least **20 years**. Make sure that the root CA used to sign the subordinate CA can support this validity period.

Reviewer 2: ```suggestion
The subordinate CA certificate must be valid for at least **20 years**. Make sure that the root CA used to sign the subordinate CA certificate supports this validity period.
```

> Follow the steps below:

Reviewer 2: ```suggestion
Perform the following steps:
```
